components/camel-weka/src/main/java/org/apache/camel/component/weka/WekaEndpoint.java

```diff
@@ -16,22 +16,46 @@
  */
 package org.apache.camel.component.weka;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.URL;
+import java.nio.file.Paths;
+
 import org.apache.camel.Consumer;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
 import org.apache.camel.Processor;
 import org.apache.camel.Producer;
+import org.apache.camel.component.file.GenericFile;
 import org.apache.camel.spi.UriEndpoint;
 import org.apache.camel.spi.UriParam;
 import org.apache.camel.support.DefaultEndpoint;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.nessus.weka.AssertState;
+import io.nessus.weka.Dataset;
+import io.nessus.weka.ModelLoader;
+import io.nessus.weka.ModelPersister;
+import io.nessus.weka.UncheckedException;
+import weka.classifiers.Classifier;
+import weka.classifiers.Evaluation;
+import weka.core.Instances;
 import weka.core.Version;
+import weka.core.converters.ArffLoader;
+import weka.core.converters.CSVLoader;
+import weka.core.converters.Loader;
 
 /**
  * The camel-weka component provides Data Mining functionality through Weka.
  */
-@UriEndpoint(firstVersion = "3.1.0", scheme = "weka", title = "Weka",
-        syntax = "weka:cmd?options", producerOnly = true, label = "data mining")
+@UriEndpoint(firstVersion = "3.1.0", scheme = "weka", title = "Weka", syntax = "weka:cmd?options", label = "data mining")
```

Comment:
No ?option in syntax. And label should be a single word, datamining or something as a category label.

Reply:
Done